.github/workflows/compile_changelogs_fulpstation.yml

```diff
@@ -0,0 +1,73 @@
+name: Compile changelogs (FULP)
```

Comment:
What does this do that makes it worth doing this over a tg edit?

Reply:
Nothing in particular, besides avoiding the edit. Would you rather I just edit compile_changelogs?

Reply:
Replaced it with a TG edit.